src/main/charts/bamboo-agent/Changelog.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## 1.7.0
```

Comment:
Do we added a new feature in this release? I see adding some env variable for os query but it does not looks a new feature to me. In this case we may pick `1.6.1` for the release version instead. WDYT?

Reply:
The `forceConfigUpdate` flag is new to the charts, so bumping the minor version seems appropriate.